File: 2016-maximum-difference-between-increasing-elements/2016-maximum-difference-between-increasing-elements.java
class Solution {
    public int maximumDifference(int[] nums) {
        int ans = -1;
        for (int i = 0; i < nums.length - 1; i++) {
            for (int j = i+1; j < nums.length; j++) {
                if (nums[i] < nums[j]) {
                    int temp = nums[j] - nums[i];
                    ans = Math.max(ans, temp);
                }else{
                    ans=ans;
                }
            }
        }
        return ans;
    }
}

File: 2016-maximum-difference-between-increasing-elements/2016-maximum-difference-between-increasing-elements.py
class Solution:
    def maximumDifference(self, nums: List[int]) -> int:
        n=len(nums)
        low=nums[0]
        ans=-1
        for j in range(1,n):
                    if low < nums[j]:
                        temp=nums[j]-low
                        ans=max(ans,temp)
                    low=min(low,nums[j])
        return ans